data/json/npcs/NC_CARAVAN_GUARD.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "type": "item_group",
+    "id": "NC_CARAVAN_GUARD_pants",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "jeans", "prob": 20 },
+      { "item": "pants", "prob": 20 },
+      { "item": "pants_leather", "prob": 20 },
+      { "item": "pants_cargo", "prob": 60 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_CARAVAN_GUARD_shoes",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "sneakers", "prob": 20 },
+      { "item": "boots", "prob": 20 },
+      { "item": "boots_steel", "prob": 40 },
+      { "item": "sneakers_steel", "prob": 28 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_CARAVAN_GUARD_coat",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "null", "prob": 20 },
+      { "item": "hoodie", "prob": 20 },
+      { "item": "jacket_light", "prob": 20 },
+      { "item": "jacket_windbreaker", "prob": 10 },
+      { "item": "jacket_leather", "prob": 80 },
+      { "item": "trenchcoat", "prob": 20 },
+      { "item": "jacket_jean", "prob": 20 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_CARAVAN_GUARD_vest",
+    "subtype": "distribution",
+    "entries": [ { "item": "null", "prob": 20 }, { "item": "kevlar", "prob": 80 }, { "item": "tacvest", "prob": 40 } ]
+  },
```

Comment:
Disclaimer, I'm not super savvy on what this is interpreted as and handled in the code. Is it the case between these null probabilities that there is a 4% chance caravan guards are topless?

If so, my suggestion is to replace either null result with some plain (non-protective) clothes item, if the intent is that they be unarmored.

Reply:
Thanks, @MNG-cataclysm has changed it to be a t-shirt in so have committed that